pubtools/iib/iib_ops.py

```diff
@@ -143,6 +142,12 @@
     "required": False,
     "type": str,
 }
+ADD_CMD_ARGS[("--check-related-images",)] = {
```

Comment:
MIchal, I tested the PR against IIB QE, when I try to run pubtools-iib-add-bundle command, I am getting "error: unrecognized arguments: --check-related-images
"

Reply:
I found out, that iiblib needs to be released first, then we can merge this, and also continue with a release of pubtools-iib.

IIBLIB PR blocking this: https://github.com/release-engineering/iiblib/pull/65

Reply:
The problem should no longer be present.